en/scale-a-tidb-cluster.md

```diff
@@ -1,60 +1,74 @@
+
+1. Modify the `replicas` value of a component as needed. For example, running the following command can dynamically configure the `replicas` value of PD to 3:
 
     {{< copyable "shell-regular" >}}
 
     ```shell
-    kubectl edit tidbcluster ${cluster_name} -n ${namespace}
+    kubectl patch -n ${namespace} tc ${cluster_name} --type merge --patch '{"spec":{"pd":{"replicas":3}}}'
     ```
 
-Check whether the TiDB cluster in Kubernetes has updated to your desired definition by running the following command:
+2. Check whether your configuration has been updated in the corresponding TiDB cluster in Kubernetes.
 
-{{< copyable "shell-regular" >}}
+    {{< copyable "shell-regular" >}}
 
-```shell
-kubectl get tidbcluster ${cluster_name} -n ${namespace} -oyaml
-```
+    ```shell
+    kubectl get tidbcluster ${cluster_name} -n ${namespace} -oyaml
+    ```
 
-In the `TidbCluster` file output by the command above, if the values of `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` are consistent with the values you have modified, check whether the number of `TidbCluster` Pods has increased or decreased by running the following command:
+    In the `TidbCluster` file output by the command above, the values of `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` are expected to be consistent with the values you have configured.
 
-{{< copyable "shell-regular" >}}
+3. Check whether the number of `TidbCluster` Pods has increased or decreased.
 
-```shell
-watch kubectl -n ${namespace} get pod -o wide
-```
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    watch kubectl -n ${namespace} get pod -o wide
+    ```
 
-For the PD and TiDB components, it might take 10-30 seconds to scale in or out.
+    For the PD and TiDB components, it might take 10-30 seconds to scale in or out.
 
-For the TiKV component, it might take 3-5 minutes to scale in or out because the process involves data migration.
+    For the TiKV component, it might take 3-5 minutes to scale in or out because the process involves data migration.
 
-#### Scale out TiFlash
+### Horizontally scale TiFlash
 
-If TiFlash is deployed in the cluster, you can scale out TiFlash by modifying `spec.tiflash.replicas`.
+This section describes how to horizontally scale out or scale in TiFlash if you have deployed TiFlash in the cluster.
 
-#### Scale TiCDC
+#### Horizontally scale out TiFlash
 
-If TiCDC is deployed in the cluster, you can scale out TiCDC by modifying `spec.ticdc.replicas`.
+To scale out TiFlash horizontally, you can modify `spec.tiflash.replicas`.
 
-#### Scale in TiFlash
+For example, running the following command can dynamically configure the `replicas` value of TiFlash to 3:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl patch -n ${namespace} tc ${cluster_name} --type merge --patch '{"spec":{"tiflash":{"replicas":3}}'
```

Comment:
```suggestion
kubectl patch -n ${namespace} tc ${cluster_name} --type merge --patch '{"spec":{"tiflash":{"replicas":3}}}'
```

Reply:
@DanielZhangQD Please confirm.

Reply:
@ran-huang I think it's a typo because @KanShiori added several commands like this one and in some other command three `}` is used.